reference/configuration/framework.rst

```diff
@@ -1187,7 +1189,94 @@ individually for each asset package:
+       framework:
+            assets:
+                # this manifest is applied to every asset (including packages)
+                json_manifest_path: "%kernel.root_dir%/../web/assets/manifest.json"
+                packages:
+                    foo_package:
+                        # this package uses its own manifest (the default file is ignored)
+                        json_manifest_path: "%kernel.root_dir%/../web/assets/a_different_manifest.json"
+                    baz_package:
+                        # this package uses the global manifest (the default file is used)
+                        base_path: '/images'
+
+    .. code-block:: xml
+
+        <!-- app/config/config.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+                <framework:assets json_manifest_path="%kernel.root_dir%/../web/assets/manifest.json">
+                    <!-- this package removes the manifest (the file will not apply) -->
+                    <framework:package
+                        name="foo_package"
+                        json_manifest_path="%kernel.root_dir%/../web/assets/a_different_manifest.json" />
+                    <!-- this package uses the global manifest (the default file is used) -->
+                    <framework:package
+                        name="baz_package"
+                        base_path="/images" />
+                </framework:assets>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // app/config/config.php
+        $container->loadFromExtension('framework', array(
+            'assets' => array(
+                'version_strategy' => '%kernel.root_dir%/../web/assets/manifest.json',
+                'packages' => array(
+                    'foo_package' => array(
+                        // this package uses its own manifest (the default file is ignored)
+                        'json_manifest_path' => '%kernel.root_dir%/../web/assets/a_different_manifest.json',
+                    ),
+                    'baz_package' => array(
+                        // this package uses the global manifest (the default file is used)
+                        'json_manifest_path' => '/images',
+                    ),
+                ),
+            ),
+        ));
+
+.. note::
+
+    This parameter cannot be set at the same time as ``version`` or ``version_strategy``.
+    Additionally, parameter cannot be nullified at the package scope if a global manifest
+    file is specified.
```

Comment:
Maybe add a new sentence (probably not in the note, but at the bottom somewhere):

> If an asset's path is not found in the manifest file, the original - unmodified - path will be used.